src/main/java/org/jabref/gui/groups/GroupTreeViewModel.java

```diff
@@ -386,7 +396,6 @@ public void editGroup(GroupNodeViewModel oldGroup) {
     }
 
     public void chatWithGroup(GroupNodeViewModel group) {
-        // This should probably be done some other way. Please don't blame, it's just a thing to make it quick and fast.
         if (currentDatabase.isEmpty()) {
             dialogService.showErrorDialogAndWait(Localization.lang("Unable to chat with group"), Localization.lang("No library is selected."));
```

Comment:
Is this branch ever it in th execution? Maybe, it can be replaced by a `LOGGER.warn` statement?

Reply:
Well, the code really can't proceed without a database.

I don't like that if there is no database, there is no reasonably visible response from the JabRef. Just a LOGGER.warn. Not sure what should I do